<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Browser1DPlotWidget" name="plot_widget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="plot_controls_layout">
     <item>
      <widget class="QCheckBox" name="autorange_checkbox">
       <property name="text">
        <string>Auto-range</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="autorange_button">
       <property name="text">
        <string>Reset View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cursor_button">
       <property name="text">
        <string>Center Cursor</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="symbol_checkbox">
         <property name="text">
          <string>Markers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="symbol_combobox">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Mouse:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hover_coords_label">
       <property name="text">
        <string>(252, 119)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Browser1DPlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>firefly.run_browser.lineplot_view</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>symbol_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>symbol_combobox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>643</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
